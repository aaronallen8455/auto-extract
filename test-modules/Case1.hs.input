module Case1 where

test = EXTRACT@simple True
