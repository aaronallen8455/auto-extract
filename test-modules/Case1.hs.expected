module Case1 where

test = simple

simple :: Bool
simple = True
